_inc/client/security/private.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { FEATURE_PRIVATE_JETPACK } from 'lib/plans/constants';
+import { ModuleToggle } from 'components/module-toggle';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { withModuleSettingsFormHelpers } from 'components/module-settings/with-module-settings-form-helpers';
+
+export const Private = withModuleSettingsFormHelpers(
+	class extends Component {
+		render() {
+			const module_enabled = this.props.getOptionValue( 'private' );
+			return (
+				<SettingsCard
+					{ ...this.props }
+					module="private"
+					feature={ FEATURE_PRIVATE_JETPACK }
+					hideButton
+				>
+					<SettingsGroup
+						hasChild
+						module={ { module: 'private' } }
+						support={ {
+							text: __(
+								"This option is great if you're still working on your site and aren't quite ready to show it off to the rest of the internet yet, or if you're publishing things you'd like to keep private or make available only to specific people. You can make your site public whenever you'd like (or vice versa)."
```

Comment:
I've reduced the text in [0f9e51b](https://github.com/Automattic/jetpack/pull/10333/commits/0f9e51b8f7ce9cfc1aecabcbdcefd88281d800fc). Since we are linking to `Learn more`, we'll have an opportunity to give more details in the support page.